File: Exercism Codes/Scala/Hamming.scala
object Hamming {
  def distance(dnaStrandOne: String, dnaStrandTwo: String): Option[Int] = {
    if(dnaStrandOne.length != dnaStrandTwo.length) return None
    Some(Range(0, dnaStrandOne.length).count(i => dnaStrandOne(i) != dnaStrandTwo(i)))
  }
}

File: Exercism Codes/C#/Hamming.cs
using System;
using System.Linq;
public static class Hamming
{
    public static int Distance(string firstStrand, string secondStrand) => firstStrand.Length != secondStrand.Length
        ? throw new ArgumentException()
        : Enumerable.Range(0, firstStrand.Length).Where(x => firstStrand[x] != secondStrand[x]).Count();
}